packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -78,12 +84,20 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
         @inject(WidgetManager) protected readonly widgetManager: WidgetManager,
         @inject(FileSystem) protected readonly fileSystem: FileSystem,
         @inject(SelectionService) protected readonly selectionService: SelectionService,
+        @inject(QuickPickService) protected readonly quickPick: QuickPickService,
```

Comment:
new dependendencies should be property dependencies to avoid breaking changes of a constructor